Subject: drop lcrypto from NSC_LNETSNMPLIBS

The -lcrypto in NSC_LNETSNMPLIBS shouldn't be required for most compiles. It
will break static linking but usually don't do this.
The main reason for this is to avoid pullin in libssl's dev package in.

Signed-of-by: Sebastian Andrzej Siewior <sebastian@breakpoint.c>
---
 net-snmp-config.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/net-snmp-config.in
+++ b/net-snmp-config.in
@@ -49,7 +49,7 @@ datarootdir=@datarootdir@
 NSC_LDFLAGS="@LDFLAGS@"
 
 NSC_LIBS="@LIBS@"
-NSC_LNETSNMPLIBS="@LNETSNMPLIBS@"
+NSC_LNETSNMPLIBS="" #"@LNETSNMPLIBS@"
 NSC_LAGENTLIBS="@LAGENTLIBS@ @PERLLDOPTS_FOR_APPS@"
 NSC_LMIBLIBS="@LMIBLIBS@"
 
